lib/plugins/create/create.test.js

```diff
@@ -228,6 +228,33 @@ describe('Create', () => {
       });
     });
 
+    it('should generate scaffolding for "aws-kotlin-node" template', () => {
+      process.chdir(tmpDir);
+      create.options.template = 'aws-kotlin-node';
+
+      return create.create().then(() => {
+        const dirContent = walkDirSync(tmpDir)
+          .map(elem => elem.replace(path.join(tmpDir, path.sep), ''));
+
```

Comment:
It looks like the check for `package.json` is missing here.

Reply:
Yep, fix incoming.

Reply:
Thanks!